Documentation/feature-band-source-building.md

```diff
@@ -0,0 +1,825 @@
+
+1. **Stage 1**: Use Microsoft-provided source-built artifacts and Microsoft
+   SDK to build the target release
+2. **Stage 2**: Use the outputs from Stage 1 to rebuild the same release from
+   source
+
+The second stage validates that the release can be built using only
+source-built components and produces the final artifacts for distribution.
+
+```mermaid
+flowchart TD
+    subgraph Bootstrap["Bootstrap (Two-Stage)"]
+        MS_SDK[Microsoft SDK] --> Stage1[Stage 1 Build]
+        MS_Artifacts[Microsoft Source-Built<br/>Artifacts] --> Stage1
+        Stage1 --> SB_SDK[Source-Built SDK]
+        Stage1 --> SB_Artifacts[Source-Built Artifacts]
+        SB_SDK --> Stage2[Stage 2 Build]
+        SB_Artifacts --> Stage2
+        Stage2 --> Final1[Final Source-Built SDK]
+        Stage2 --> Final2[Final Source-Built<br/>Artifacts]
+    end
+```
+
+See [Bootstrapping Guidelines](bootstrapping-guidelines.md) for more info.
+
+### Sequential Build (Single-Stage)
+
+When you have appropriate source-built artifacts from previous builds, you can
+build directly without bootstrapping. This is the typical workflow for ongoing
+releases.
+
+```mermaid
+flowchart TD
+    subgraph Sequential["Sequential Build (Single-Stage)"]
+        Prev_SDK[Previous Source-Built SDK] --> Build
+        Prev_Artifacts[Previous Source-Built<br/>Artifacts] --> Build
+        Build --> Final1[Final Source-Built SDK]
+        Build --> Final2[Final Source-Built<br/>Artifacts]
+    end
+```
+
+## Required Build Inputs
+
+When building feature bands, you'll work with two types of source-built
+artifacts:
+
+### Previously Source-Built (PSB) Artifacts
+
+These are artifacts from previous servicing releases or bootstrap builds:
+
+- Used as build dependencies but not redistributed in outputs
+- Come from previous monthly .NET releases (e.g., 8.0.1 artifacts when
+  building 8.0.2)
+- Subject to [poison detection](leak-detection.md) to ensure no Microsoft
+  binaries leak into outputs
+
+### Shared Component Artifacts
+
+These are shared runtime and foundational components from the 1xx band:
+
+- Come from the 1xx band build in the same release cycle
+- Can be used as dependencies AND redistributed in outputs
+- Include shared runtime and foundational components that all feature bands
+  depend on
+
+## Feature Band Characteristics
+
+### 1xx Band (GA/Non-breaking)
+
+The foundational feature band that ships with .NET GA releases:
+
+- **Content**: Contains the complete VMR with .NET runtime, core libraries,
+  and SDK tooling
+- **Purpose**: Provides the stable runtime foundation that all other bands
+  depend on
+- **Release cycle**: Follows .NET GA release schedule with monthly servicing
+  updates
+- **Compatibility**: Non-breaking changes only during servicing
+
+### 2xx Band (Stable/Enhanced)
+
+Enhanced SDK tooling that ships with Visual Studio updates:
+
+- **Content**: Contains only SDK tooling differences from 1xx band (subset
```

Comment:
I'm not fond of including "differences" here.  It can be interpreted that source deltas are the only thing contained.  The entire SDK tooling is included not just the differences.